src/cli/init/prompts.ts

```diff
@@ -76,6 +76,7 @@ export const TEMPLATE_PROMPT = new Select({
   choices: [
     'greeter',
     'koa-rest-api',
+    'express-rest-api',
```

Comment:
Would you mind alphabetically sorting this here, in the README, etc? It means you get top billing 🙂